src/main/java/com/fortanix/sdkms/performance/sampler/AbstractSignatureSampler.java

```diff
@@ -43,6 +42,11 @@ public void setupTest(JavaSamplerContext context) {
         String keySize = context.getParameter(KEY_SIZE, "1024");
         String filePath = context.getParameter(FILE_PATH);
         int batchSize = context.getIntParameter(BATCH_SIZE, 0);
+        String hashAlorithm = context.getParameter(HASH_ALGORITHM);
+        DigestAlgorithm digestAlgorithm = DigestAlgorithm.fromValue(hashAlorithm.toUpperCase());
```

Comment:
This will have Nullpointer exception when `hashAlorithm` is not set. you should set a default value to context fetching code: `context.getParameter(HASH_ALGORITHM, "SHA1");`

Reply:
fixed